plexpy/webserve.py

```diff
@@ -816,11 +817,17 @@ def get_library_watch_time_stats(self, section_id=None, grouping=None, **kwargs)
                      ]
             ```
         """
+        if query_days:
+            query_days = map(int, query_days.split(","))
+        else:
+            # Default values for backwards compatibility
+            query_days = [1, 7, 30, 0]
```

Comment:
Can leave out the `else` block since the default is added to `get_watch_time_stats`.